src/main/java/org/duckstudy/movingcar/CarMover.java

```diff
@@ -0,0 +1,26 @@
+package org.duckstudy.movingcar;
+
+import java.util.List;
+
+public class CarMover {
+    private final RandomValueGenerator randomValueGenerator;
+
+    public CarMover(RandomValueGenerator randomValueGenerator) {
+        this.randomValueGenerator = randomValueGenerator;
+    }
+
+    public void move(Car car) {
+        int randomValue = randomValueGenerator.generateRandomValue();
+        if (randomValue >= 4) {
+            car.move();
+        }
+    }
+
+    public void move(List<Car> carList) {
+        for (Car car : carList) {
+            this.move(car);
```

Comment:
```suggestion
            move(car);
```
this를 생략할때와 명시할때의 명지님의 기준이 있을까요?

Reply:
this를 사용한 이유는 move 함수가 오버로딩 되어서 그 부분을 명확하게 보여주고 싶어서 작성하였는데요,
그런데 다시 생각해보니 this를 굳이 사용하지 않아도 충분히 어떤 메서드를 호출하는지 알 것 같아서 굳이 this를 사용하지 않아도 될 것 같아요 !